tests/acceptance/lib/files/manage_variable_values_ini.cf.start

```diff
@@ -1,5 +1,9 @@
 [section]
 keyone=valueone
 keytwo=value2
+[CFE-3221]
+variablename= desired value
+variablename2 = desired value2
```

Comment:
These lines in the managed section have spaces after the = when the test starts.